src/shared/http/Curl.php

```diff
@@ -31,10 +31,10 @@
  * @codeCoverageIgnore
  */
 class Curl {
-    /** @var CurlHandle|resource */
+    /** @var CurlHandle */
```

Comment:
This is the same "problem" as with the phpstan related change. This is only true for PHP 8+, PHP 7.x *does* uses `resource` ...

Reply:
You're right, I faced the same issue than with PHPStan, except Psalm doesn't tolerate the curl functions accept CurlHandle or a resource at the same time, so I had to choose one.

Here another possibility that works too:
```
/**
 * @var CurlHandle|resource
 * @psalm-var CurlHandle
 */
```
Would you prefer this solution?

Reply:
I'm not sure what would be the best choice. For the time being, I want `phive` to work with PHP 7.3+, maybe PHP 7.4+ so we cannot make any changes as of yet that work with PHP 8+ only.

If that makes psalm and/or phpstan unhappy, we have to configure exceptions for them.